content/04.results.md

```diff
@@ -157,46 +157,22 @@ The number of SV and CNV breakpoints were significantly correlated across tumors
-These tumors are divided into mature and immature (malignant) groups, where pediatric tumors are mostly classified as mature ones [@doi: 10.1016/j.jpedsurg.2004.03.045]. 
-Mature tumors are deficient of malignant potential thereby reducing the risk of malignancy [@doi: 10.1016/j.jpedsurg.2004.03.045], hence bearing low telomerase activities.
-Benign neoplasia including dysplasia and hemangioblastomas had the lowest scores. 
-Among common pediatric brain tumors, medulloblastomas had higher scores than high grade astrocytic tumors, ependymomas and low-grade astrocytomas. 
-Across broad groups, benign tumors and precancerous lesions were among those with the lowest scores, whereas pineal region and embryonal tumors have the highest scores (Figure) {@fig:distribution-plots}B. 
-Considering medulloblastoma tumors, high risk Group3 and Group4 subtypes had higher scores than WNT and SHH subtype (Figure) {@fig:distribution-plots}C. 
-A similar pattern has been speculated with respect to cell of origin by one of the earlier studies [@doi:10.1007/s00401-013-1198-2] which mainly focused TERT promoter mutations for medulloblastoma subtypes comparisons rather than telomerase activity.
-
-![Box plots representing distribution of telomerase activities for stranded FPKM data.
-A) short histological classifications.
-B) broad histological classifications.
-C) Comparison of telomerase activities across molecular subtypes of medulloblastoma tumors. P-values have been calculated using series of two-sided t-tests and corrected via Bonferroni correction.
-](https://github.com/AlexsLemonade/OpenPBTA-analysis/raw/master/figures/pngs/Telomerase_Activities.png){#fig:distribution-plots}
+#### Histologic and oncogenic pathway clustering
+
+We performed UMAP unsupervised dimension reduction to validate that brain tumors cluster by histology (**Figure {@fig:Fig4}A**) and geneset variant analysis (GSVA) for Hallmark cancer gene sets to demonstrate activation of underlying oncogenic pathways (**Figure {@fig:Fig4}B**).	
+We observed medulloblastoma samples cluster by molecular subtype with Groups 3 and 4 tightly clustered and WNT and SHH in distinct clusters (**Figure {@fig:FigS4}A**), as expected.
+Additionally, _C11orf95::RELA_ fusion positive ependymomas and _BRAF_-driven low-grade tumors formed distinct clusters.
+
+#### Telomerase activity 
+
+We next used gene expression data to predict telomerase activity using EXpression-based Telomerase ENzymatic activity Detection (EXTEND) [@doi:10.1038/s41467-020-20474-9] as a surrogate measure of malignant potential [@doi:10.1038/s41467-020-20474-9; 10.1093/carcin/bgp268].
+EXTEND scores calculated using FPKM or counts were tightly correlated (p < 2.2e-16) for each library type (**Figure {@fig:FigS4}E-F**) and significantly correlated with _TERT_ and _TERC_ expression (**Figure {@fig:FigS4}G-J**).  
```

Comment:
I feel like this sentence, while a good check, may not need to be in the main body. In some sense this is the expected recapitulation of the EXTEND paper?

Reply:
yeah, I think this was written before the telomerase paper came out, so I tried to keep some of the original text, but you are right - I will shorten.